File: MPP/Curs3/Curs3/TasksSpringAutowire/src/main/java/tasks/repository/RepositoryException.java
package tasks.repository;

/**
 * Created by grigo on 11/14/16.
 */
public class RepositoryException extends RuntimeException {
    public RepositoryException(String message) {
        super(message);
    }

    public RepositoryException(Exception ex) {
        super(ex);
    }
}

File: MPP/ClientServer_C#/Persistence/utils/RepositoryException.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Concurs.repository.utils
{
    public class RepositoryException : ApplicationException
    {
        public RepositoryException() { }
        public RepositoryException(String mess) : base(mess) { }
        public RepositoryException(String mess, Exception e) : base(mess, e) { }
    }
}
